boa_engine/src/builtins/mod.rs

```diff
@@ -369,6 +387,12 @@ pub(crate) fn set_default_global_bindings(context: &mut Context<'_>) -> JsResult
     #[cfg(feature = "intl")]
     global_binding::<intl::Intl>(context)?;
 
+    #[cfg(feature = "experimental")]
+    {
+        global_binding::<temporal::Temporal>(context)?;
+        global_binding::<temporal::TimeZone>(context)?;
```

Comment:
This adds a `TimeZone` property to the global object, but the spec only allows calling the constructor with `new Temporal.TimeZone`. Only adding `Temporal` to the global object should be enough.